examples/mobile/UIComponents/components/controllers/chips-add.js

```diff
@@ -0,0 +1,32 @@
+/* global tau */
+(function () {
+	var page = document.getElementById("chips-add-page"),
+		input = document.getElementById("chip-content"),
```

Comment:
I think it would good to use the 'page.querySelector' instead of the 'document.getElementById'
```
page.querySelector('#chip-content')
```
It creates the possibility of finding the element outside the page.

Reply:
Done.